server/api.js

```diff
@@ -748,6 +752,13 @@ router.post("/bid", async (req, res) => {
 
 			const bidResult = await client.query(bidQuery, bidValues);
 
+			const updateTenderQuery = `
+				UPDATE tender
+				SET no_of_bids_received = no_of_bids_received + 1
+				WHERE id = $1
+			`;
```

Comment:
This doesn't belong here. Move it to another PR

Reply:
This has already been merged. so I made a mistake because I opened a branch on top of the previous branch with this commit that has already been merged before this